### PR TITLE
Mandatory package meta warning added

### PR DESCRIPTION
**Pull request by Karina5005**
https://github.com/objectionary/eo/issues/511

### CODE REVIEW

File: eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            <xsl:attribute name="check">
+              <xsl:text>wrong-package</xsl:text>
+            </xsl:attribute>
+            <xsl:attribute name="line">
+              <xsl:value-of select="@line"/>
+            </xsl:attribute>
+            <xsl:attribute name="severity">
+              <xsl:text>warning</xsl:text>
+            </xsl:attribute>
+            <xsl:text>Wrong package</xsl:text>
```
Comment (yegor256) [inline eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl:41]: @Karina5005 I think this message is a bit misleading. The errors is not about the "wrong" package, but about the package meta that is absent. Would be great to make the text more verbose and detailed

File: eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" id="package-meta" version="2.0">
+  <xsl:output encoding="UTF-8"/>
+  <xsl:template match="/program/errors">
+    <xsl:copy>
+      <xsl:apply-templates select="node()|@*"/>
+      <xsl:if test="count(/program/metas/meta[head ='package'and count(part)=1])!=1">
```
Comment (yegor256) [inline eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl:30]: @Karina5005 maybe we can create TWO types of warnings in this XSL. First, when +package is absent. Second, when there is more than one +package.

**Comment by yegor256** [comment]
@Karina5005 see some comments above

File: eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl (hunk 2 above)
Comment (Karina5005) [reply eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl:30]: Okay, but I think it's better to break it down into three cases, there's still the possibility that the package name is incorrect/missing.

Comment (Karina5005) [reply eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl:30]: @yegor256 am i right?

Comment (yegor256) [reply eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl:30]: @Karina5005 yes, agree, but I would create another XSL for the third case

File: eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl (hunk 1 above)
Comment (Karina5005) [reply eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl]: Fixed

File: eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl (hunk 2 above)
Comment (Karina5005) [reply eo-parser/src/main/resources/org/eolang/parser/warnings/mandatory-package-meta.xsl]: Done

**Comment by Karina5005** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@rultor merge

**Comment by Karina5005** [comment]
@yegor256 I fixed style according to xcop, it should be fine now.

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@Karina5005 thanks!

**Pull request merged**